packages/ui/src/components/Notifications/NotificationItem.tsx

```diff
@@ -0,0 +1,70 @@
+import { FC } from "react"
+
+interface NotificationItemProps {
+  action: string
+  description: string
+  isUnread?: boolean
+  time: string
+}
+
+const NotificationItem: FC<NotificationItemProps> = ({
+  action,
+  description,
+  isUnread,
+  time,
+}) => {
+  return (
+    <button
+      onClick={() => console.log("TODO")}
```

Comment:
@vladutjs dapps should pass the click impl or do we want to redirect to starkscan (or other)?